src/CRKSPH/CRKSPH.cc

```diff
@@ -89,6 +89,15 @@ CRKSPH(DataBase<Dimension>& dataBase,
   mPairAccelerationsPtr() {
 }
 
+//------------------------------------------------------------------------------
+// Destructor.
+//------------------------------------------------------------------------------
+template<typename Dimension>
+CRKSPH<Dimension>::
+~CRKSPH(){
+  // Needs to be here due to implicit PairwiseField delete
+}
+
```

Comment:
Had to add this to all the hydro packages. Otherwise, the build fails since the PairwiseField destructor isn't available in the header file.